topology/src/main/resources/proto/proto.lock

```diff
@@ -0,0 +1,549 @@
+{
+  "definitions": [
+    {
+      "protopath": "requests.proto",
```

Comment:
:thought_balloon: Do we want to / need to keep backwards compatibility here? The requests are ephemeral and it should be fine for scaling not to work during rolling updates, right?

Reply:
By default, let's try not to keep the compatibility. When we have to break compatibility, we can do that deliberately.